RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,476 @@
+        Label
+    </Button>
+    <Button
+        mode="contained-tonal"
+        buttonColor={theme.colors.errorContainer}
+        textColor={theme.colors.onErrorContainer}
+    >
+        Label
+    </Button>
+    <Button mode="contained-tonal" disabled>
+        Label
+    </Button>
+    ```
+
+- ###  Data Table
+
+    Recommended style overrides:
+    * Text color of Table Title: onSurface
+    * font size of Table Title: theme.fonts.labelLarge
+
+    <img width='300' alt="Data Table" src="./Images/dataTable.png"><br/>
+
+    Usage
+    ```tsx
+    import { DataTable } from 'react-native-paper';
+
+    <DataTable>
+        <DataTable.Header>
+            <DataTable.Title
+                textStyle={{ color: theme.colors.onSurface, ...theme.fonts.labelLarge }}
+            >
+                Dessert
+            </DataTable.Title>
+            <DataTable.Title
+                numeric
+                textStyle={{ color: theme.colors.onSurface, ...theme.fonts.labelLarge }}
+            >
+                Calories
+            </DataTable.Title>
+            <DataTable.Title
+                numeric
+                textStyle={{ color: theme.colors.onSurface, ...theme.fonts.labelLarge }}
+            >
+                Fat
+            </DataTable.Title>
+        </DataTable.Header>
+
+        {items.slice(from, to).map((item) => (
+            <DataTable.Row key={item.key}>
+                <DataTable.Cell>{item.name}</DataTable.Cell>
+                <DataTable.Cell numeric>{item.calories}</DataTable.Cell>
+                <DataTable.Cell numeric>{item.fat}</DataTable.Cell>
+            </DataTable.Row>
+        ))}
+
+        <DataTable.Pagination
+            page={page}
+            numberOfPages={Math.ceil(items.length / itemsPerPage)}
+            label={`${from + 1}-${to} of ${items.length}`}
+            numberOfItemsPerPageList={numberOfItemsPerPageList}
+            numberOfItemsPerPage={itemsPerPage}
+            onItemsPerPageChange={onItemsPerPageChange}
+            showFastPaginationControls
+            selectPageDropdownLabel={'Rows per page'}
+        />
+    </DataTable>
+    ```
+
+- ###  Icon button
+
+    Recommended style overrides:
+    * background color for contained mode: primary
+    * icon color for contained mode: onPrimary
+    * background color for contained-tonal mode: primaryContainer
+    * icon color for contained-tonal mode: onPrimaryContainer
+
+    <img width='300' alt="IconButton" src="./Images/iconButton.png"><br/>
+
+    Usage
+    ```tsx
+    import { IconButton } from 'react-native-paper';
+
+    <IconButton icon="camera"/>
+    <IconButton icon="video" mode='outlined'/>
+    <IconButton icon="camera" mode='contained' containerColor={theme.colors.primary} iconColor={theme.colors.onPrimary} />
+    <IconButton icon="video" mode='contained-tonal' containerColor={theme.colors.primaryContainer} iconColor={theme.colors.onPrimaryContainer}  />
+    ```
+
+- ###  ProgressBar
```

Comment:
Either make it `Progress Bar` (with a space in the middle) or make everything else upper camel case like what you have here